test/examples/ucp_hello_world.c

```diff
@@ -507,6 +485,154 @@ int main(int argc, char **argv)
     ucp_cleanup(ucp_context);
 
 err:
-
     return ret;
 }
+
+int parse_cmd(int argc, char * const argv[], char **server_name)
+{
+    int c = 0, index = 0;
+    opterr = 0;
+    while ((c = getopt (argc, argv, "pgrn:h")) != -1) {
+        switch (c) {
+        case 'p':
```

Comment:
pls also add option to select TCP port number, and pass different port number for each executor in jenkins script.
For example:
`PORT=$((10000 + $EXECUTOR_NUMBER))`